deploy/v1.6.0/optimism.ts

```diff
@@ -0,0 +1,202 @@
+import { dim } from 'chalk';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import {
+  DRAW_BUFFER_CARDINALITY,
+  PRIZE_DISTRIBUTION_BUFFER_CARDINALITY,
+  PRIZE_DISTRIBUTION_FACTORY_MINIMUM_PICK_COST,
+  TOKEN_DECIMALS,
+} from '../../src/constants';
+import { deployAndLog } from '../../src/deployAndLog';
+import { setPrizeStrategy } from '../../src/setPrizeStrategy';
+import { setTicket } from '../../src/setTicket';
+import { transferOwnership } from '../../src/transferOwnership';
+import { setManager } from '../../src/setManager';
+import { initPrizeSplit } from '../../src/initPrizeSplit';
+
+export default async function deployToOptimism(hardhat: HardhatRuntimeEnvironment) {
+  if (process.env.DEPLOY === 'v1.6.0.optimism') {
+    dim(`Deploying: Receiver Chain Optimism Mainnet`);
+    dim(`Version: 1.6.0`);
+  } else {
+    return;
+  }
+
+  const { getNamedAccounts, ethers, getChainId } = hardhat;
+
+  const {
+    deployer,
+    executiveTeam,
+    ptOperations,
+    aUSDC,
+    defenderRelayer,
+    aaveIncentivesController,
+    aaveLendingPoolAddressesProviderRegistry,
+  } = await getNamedAccounts();
+
+  const chainId = parseInt(await getChainId(), 10);
+
+  dim(`chainId ${chainId} `);
+  dim(`---------------------------------------------------`);
+  dim(`Named Accounts`);
+  dim(`---------------------------------------------------`);
+  dim(`deployer: ${deployer}`);
+  dim(`executiveTeam: ${executiveTeam}`);
+  dim(`ptOperations: ${ptOperations}`);
+  dim(`defenderRelayer: ${defenderRelayer}`);
+  dim(`aUSDC: ${aUSDC}`);
+  dim(`aaveIncentivesController: ${aaveIncentivesController}`);
+  dim(`aaveLendingPoolAddressesProviderRegistry: ${aaveLendingPoolAddressesProviderRegistry}`);
+  dim(`---------------------------------------------------\n`);
+  const startingBalance = await ethers.provider.getBalance((await ethers.getSigners())[0].address);
+
+  // ===================================================
+  // Deploy Contracts
+  // ===================================================
+
+  const aaveUsdcYieldSourceResult = await deployAndLog('AaveV3YieldSource', {
+    from: deployer,
+    args: [
+      aUSDC,
+      aaveIncentivesController,
+      aaveLendingPoolAddressesProviderRegistry,
+      'PTaOptUSDCY',
+      'PoolTogether aOptUSDC Yield',
```

Comment:
The name should come first.
```suggestion
      'PoolTogether aOptUSDC Yield',
      'PTaOptUSDCY',
```